src/ui/accessibility-and-localization/internationalization.md

```diff
@@ -346,6 +346,14 @@ return MaterialApp(
 
 ### Placeholders, plurals, and selects
 
+{{site.alert.note}}
+  When using VS Code, add the [arb-editor extension][].
+  This extension adds syntax hightlighting, snippets, diagnostics, and quick fixes.
+  to help edit `.arb` template files.
```

Comment:
Fix the spelling of "highlighting", keep the line <= 80 chars, and remove accidental double period:

```suggestion
  This extension adds syntax highlighting, snippets, 
  diagnostics, and quick fixes to help edit `.arb` template files.
```